### PR TITLE
SPCTR-2809 Fix for container and image broken and wrong placeholder image

### PR DESCRIPTION
**Pull request by abhishekrijal**
### Description
Fix for container and image broken and wrong placeholder image

### Types of changes

Bug fix (non-breaking change which fixes an issue)

### How has this been tested?
Tested by adding the empty image in case of default image and removing it. In frontend the image tag should not be visible when no image URL is set.

### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/image/save.js [hunk 1]
```diff
@@ -32,19 +32,21 @@ export default function Save( props ) {
...
-			sizes="(max-width: 480px) 150px"
-			src={ url }
-			alt={ alt }
-			className={ id ? `uag-image-${ id }` : null }
-			width={ width ? width : naturalWidth }
-			height={ height ? height : naturalHeight }
-			title={ title }
-			loading="lazy"
-		/>
+		url && '' !== url && (
```
Comment (RajkiranBagal) [inline src/blocks/image/save.js:35]: I think we do not need to check `'' !== url` here, as we are checking `url`

File: src/blocks/image/deprecated.js
Comment (RajkiranBagal) [inline src/blocks/image/deprecated.js:1]: Add deprecation in our newly decided format of files & also at the end of the object

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

**Review by rahulvarma722 — APPROVED** [review]

File: src/blocks/info-box/block.js [hunk 2]
```diff
@@ -34,5 +34,7 @@ registerBlockType( 'uagb/info-box', {
 	category: uagb_blocks_info.category,
 	edit: ( props ) => ( props.attributes.isPreview ? <PreviewImage image="info-box" /> : <Edit { ...props } /> ),
 	save,
+	__experimentalLabel: ( atts ) =>
+		applyFilters( 'uag_loop_data_source_label', __( 'Heading', 'ultimate-addons-for-gutenberg' ), atts ),
```
Comment (adi3890) [inline src/blocks/info-box/block.js:38]: default should be block name.
Info Box here

File: src/blocks/image/save.js (hunk 1 above)
Comment (abhishekrijal) [reply src/blocks/image/save.js:35]: Actually it was to handle both case of '' and undefined. @RajkiranBagal

File: src/blocks/image/deprecated.js
Comment (RajkiranBagal) [reply src/blocks/image/deprecated.js:1]: Check advanced heading depreciation of folder - v2.3.4

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**